rpc/openapi/openapi.yaml

```diff
@@ -2485,13 +2539,55 @@ components:
           example: 0
         result:
           required:
-            - "dataCommitment"
+            - "DataCommitment"
           properties:
-            dataCommitment:
+            DataCommitment:
               type: string
               example: "D70952032620CC4E2737EB8AC379806359D8E0B17B0488F627997A0B043ABDED"
           type: object
 
+    DataRootInclusionProofResponse:
+      type: object
+      required:
+        - "jsonrpc"
+        - "id"
+        - "result"
+      properties:
+        jsonrpc:
+          type: string
+          example: "2.0"
+        id:
+          type: integer
+          example: 0
+        result:
+          required:
+            - "Proof"
+          properties:
+            Proof:
+              required:
+                - "total"
+                - "index"
+                - "leaf_hash"
+                - "aunts"
+              properties:
+                total:
+                  type: string
+                  example: "2"
+                index:
+                  type: string
+                  example: "0"
+                leaf_hash:
+                  type: string
+                  example: "eoJxKCzF3m72Xiwb/Q43vJ37/2Sx8sfNS9JKJohlsYI="
+                aunts:
+                  type: array
+                  items:
+                    type: string
+                  example:
+                    - "eWb+HG/eMmukrQj4vNGyFYb3nKQncAWacq4HF5eFzDY="
```

Comment:
[optional] would it help to clarify that responses here are base64 encoded?

Reply:
this is taken from other proofs in the openapi file, I guess we will need to update the other ones as well if we add base64, what do you think? or we could leave this one similar to the other ones? i'm fine either way

Reply:
> if we add base64

Oh I thought this response was already base64 encoded but when I [attempt to decode it](https://go.dev/play/p/kVtWN4HuBBm), I get gibberish. If it's not base64 encoded then how is it encoded?

Reply:
It is base64, as far as I know, it's just not specified in the description.
For the gibberish, it's expected since it doesn't contain ascii characters, it's a byte array containing random data that is converted to base64. You could see the hex output if you do something like: `hex.EncodeToString(rawDecodedText)` in the playground

Reply:
ok got it. 

> it's just not specified in the description.

I think it should be but defer to you. For types and endpoints that our team has created or customized from vanilla Tendermint, I think it would help to document encoding.

> I guess we will need to update the other ones as well if we add base64

hmm I can see an argument for not adding it to the fields that we didn't modify (e.g. `TxSearchResponse`) to minimize the diff with upstream. If we really wanted the description for fields we didn't modify, we could add it upstream to cometbft and then pull changes in this repo.

None of this is blocking.

Reply:
like this: [2018e22](https://github.com/celestiaorg/celestia-core/pull/981/commits/2018e22b6b6779300382ad887e59e2140a6f9d37)?

Reply:
yea exactly! Thanks